common/components/ui/QRCode.tsx

```diff
@@ -35,6 +35,7 @@ export default class QRCode extends React.PureComponent<Props, State> {
     return (
       <img
         src={qr}
+        alt="QR Code"
```

Comment:
Should probably require a prop for this, QR code of what?

Reply:
Not sure what you mean by 'require a prop for this'. Do you mean it should just provide more context like what the QR code is for?

Reply:
I mean an `alt` prop for the QRCode component. So something like:
```
<QRCode
  data={wallet.getAddressString()}
  alt="Your Address"
/>
```
and then do
```
alt={`QR Code for ${this.props.alt}`}
```
or something.